src/js/script.js

```diff
@@ -26,11 +26,17 @@
     },
     widgets: {
       amount: {
-        input: 'input[name="amount"]',
+        input: 'input[class="amount"]',
         linkDecrease: 'a[href="#less"]',
         linkIncrease: 'a[href="#more"]',
       },
     },
+
+    cart: {
+      toggleTrigger: '#toggle-cart', 
```

Comment:
2. `toggleTrigger` powinien mieć wartość `.cart_summary`. Chcemy złapać za pierwszy element w divie o identyfikatorze `cart`
```suggestion
      toggleTrigger: '.cart__summary', 
```
Ponadto do obiekty `classNames`, który jest niżej musisz dodać nową właściwość:
```js
cart: {
      wrapperActive: 'active',
    },
```
Cały obiekt `classNames` powinien więc na tym etapie wyglądać następująco:
```js
const classNames = {
   menuProduct: {
      wrapperActive: 'active',
      imageVisible: 'active',
    },
    cart: {
      wrapperActive: 'active',
    },
 };
  ```